tests/cypress/latest/fixtures/myseedimage.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: elemental.cattle.io/v1beta1
+kind: SeedImage
+metadata:
+  name: mycustomseedimage
+  namespace: fleet-default
+spec:
+  baseImage: https://download.opensuse.org/repositories/isv:/Rancher:/Elemental:/Dev:/Teal53/media/iso/elemental-teal.x86_64.iso
```

Comment:
Sounds good for now but is it possible to use the `iso_to_test` variable at some point here? (for sure `sed` is needed like in the CLI test :D).

But clearly for another PR, just to keep this in mind.